xilem/src/lib.rs

```diff
@@ -34,8 +34,8 @@ pub struct Xilem<State, Logic, View>
 where
     View: WidgetView<State>,
 {
-    root_widget: RootWidget<View::Widget>,
-    driver: MasonryDriver<State, Logic, View, View::ViewState>,
+    pub root_widget: RootWidget<View::Widget>,
+    pub driver: MasonryDriver<State, Logic, View, View::ViewState>,
```

Comment:
I am going to change this API in #423, effectively back to your original suggestion.
But this is perfectly fine for now

Reply:
Sounds good.